vaadin-map-flow-parent/vaadin-map-flow/src/main/java/com/vaadin/flow/component/map/configuration/style/Text.java

```diff
@@ -0,0 +1,413 @@
+package com.vaadin.flow.component.map.configuration.style;
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.JsonIdentityReference;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+import com.vaadin.flow.component.map.configuration.AbstractConfigurationObject;
+import com.vaadin.flow.component.map.configuration.Constants;
+
+import java.io.Serializable;
+
+/**
+ * Text style that defines how to render texts, such as labels, on the map.
+ */
+public class Text extends AbstractConfigurationObject {
+    private String font;
+    private TextOffset offset;
+    private double scale;
+    private double rotation;
+    private boolean rotateWithView;
+    private TextAlign textAlign;
+    private TextBaseline textBaseline;
+    private Fill fill;
+    private Stroke stroke;
+    private Fill backgroundFill;
+    private Stroke backgroundStroke;
+    private double padding;
+
+    public Text() {
+        font = "13px sans-serif";
+        offset = new TextOffset(0, 10);
+        scale = 1;
+        rotation = 0;
+        rotateWithView = false;
+        textAlign = TextAlign.CENTER;
+        fill = new Fill("#333");
+        stroke = new Stroke("#fff", 3);
+        padding = 0;
+    }
```

Comment:
These are duplicated from the defaults in the connector for now. There are more default property values specified here, as we don't get the automatic fallback values that OpenLayers provides on the client-side.

The duplication could be removed if we would configure the client-side fallback text style from the server-side. I think it's OK as is for now. If / when add the feature to allow changing the default label style per map, then this would solve itself.